src/geom/SpanSet.cc

```diff
@@ -99,6 +99,72 @@ bool spansContiguous(Span const& a, Span const& b, bool compareY = true) {
                    : false;
 }
 
+/* Determine the intersection with a mask or its logical inverse
+ *
+ * spanSet - SpanSet object with which to intersect the mask
+ * mask - Mask object to be intersectedNot
+ * bitmask - bitpattern to used when deciding pixel membership in the mask
+ *
```

Comment:
This is not actual mark-up at all, this function is in an anonymous namespace, the comment block was provided strictly for future readers